app/api/route/message_api.rb

```diff
@@ -6,6 +6,8 @@ class MessageApi < Base
       optional :thread_id, type: Integer, desc: 'ID of thread'
       optional :order_by, type: Symbol, values:%i( created_at id  ), desc: 'Order of returned issues.'
       optional :order, type: Symbol, values:%i(asc desc), default: :asc
+      optional :after, types: [DateTime, Date], desc: 'Only messages after the end date or time are returned'
+      optional :before, types: [DateTime, Date], desc: 'Only messages before the start date or time are returned'
```

Comment:
Sorry for suggesting `:after` and `:before` 
I hadn't noticed we were using `:end_date` and `:start_date` already so I think we should stick with that.
https://github.com/cyclestreets/cyclescape/blob/master/app/api/route/issue_api.rb#L12-L13